src/Entities/QueryApi/DataPlatformIndexerStatuses.jsx

```diff
@@ -9,6 +12,7 @@ const [errors, setErrors] = useState("");
 const [timer, setTimer] = useState(null);
 
```

Comment:
@gabehamilton it looks like something has regressed here https://near.org/near/widget/Entities.QueryApi.DataPlatformIndexerStatuses